worlds/kh1/__init__.py

```diff
@@ -0,0 +1,315 @@
+                    ,"advanced_logic": bool(self.options.advanced_logic)
+                    ,"hundred_acre_wood": bool(self.options.hundred_acre_wood)
+                    ,"atlantica": bool(self.options.atlantica)
+                    ,"goal": str(self.options.goal.current_key)}
+        if self.options.randomize_keyblade_stats:
+            min_str_bonus = min(self.options.keyblade_min_str, self.options.keyblade_max_str)
+            max_str_bonus = max(self.options.keyblade_min_str, self.options.keyblade_max_str)
+            min_mp_bonus = min(self.options.keyblade_min_mp, self.options.keyblade_max_mp)
+            max_mp_bonus = max(self.options.keyblade_min_mp, self.options.keyblade_max_mp)
+            slot_data["keyblade_stats"] = ""
+            for i in range(22):
+                if i < 4 and self.options.bad_starting_weapons:
+                    slot_data["keyblade_stats"] = slot_data["keyblade_stats"] + "1,0,"
+                else:
+                    if min_str_bonus != max_str_bonus:
+                        str_bonus = int(self.random.randrange(min_str_bonus,max_str_bonus))
+                    else:
+                        str_bonus = int(min_str_bonus)
+                    if min_mp_bonus != max_mp_bonus:
+                        mp_bonus = int(self.random.randrange(min_mp_bonus,max_mp_bonus))
+                    else:
+                        mp_bonus = int(min_mp_bonus)
+                    slot_data["keyblade_stats"] = slot_data["keyblade_stats"] + str(str_bonus) + "," + str(mp_bonus) + ","
+            slot_data["keyblade_stats"] = slot_data["keyblade_stats"][:-1]
+        if self.options.donald_death_link:
+            slot_data["donalddl"] = ""
+        if self.options.goofy_death_link:
+            slot_data["goofydl"] = ""
+        if self.options.keyblades_unlock_chests:
+            slot_data["chestslocked"] = ""
+        else:
+            slot_data["chestsunlocked"] = ""
+        if self.options.interact_in_battle:
+            slot_data["interactinbattle"] = ""
+        return slot_data
+    
+    def create_item(self, name: str) -> KH1Item:
+        data = item_table[name]
+        return KH1Item(name, data.classification, data.code, self.player)
+
+    def create_event(self, name: str) -> KH1Item:
+        data = event_item_table[name]
+        return KH1Item(name, data.classification, data.code, self.player)
+
+    def set_rules(self):
+        set_rules(self.multiworld, self.player, self.options, self.determine_reports_required_to_open_end_of_the_world(), self.determine_reports_required_to_open_final_rest_door(), self.options.final_rest_door.current_key)
+
+    def create_regions(self):
+        create_regions(self.multiworld, self.player, self.options)
+    
+    def get_numbers_of_reports_to_consider(self) -> int:
+        numbers_to_consider = []
+        if self.options.end_of_the_world_unlock.current_key == "reports":
+            numbers_to_consider.append(self.options.required_reports_eotw)
```

Comment:
```suggestion
        if self.options.end_of_the_world_unlock.current_key == "reports":
            numbers_to_consider.append(self.options.required_reports_eotw.value)
```
Assuming the intent is for the return typing of this function to be List[int], you should add .value to the ends of each of these so that it actually gives the int instead of the option type

Reply:
Added